rescript

```diff
@@ -502,7 +503,7 @@ if (
       var event_time = +Date.now();
       var time_diff = event_time - LAST_BUILD_START;
       dlog(`Since last build : ${time_diff}`);
-      if (time_diff < 5) {
+      if (time_diff < 5 || event_time - LAST_FIRED_EVENT < 5) {
         // for 5ms, we could think that the ninja not get
         // kicked yet, so there is really no need
         // to send more events here
```

Comment:
This is another way to prevent event spamming.
For two close events fired, if one is already recorded to trigger a rebuild, the second will be ignored.
Note LAST_BUILD_START is different from this, this help filter more events ahead of time. There are cases
where a bunch of events happened while `LAST_BUILD_START` are not updated due to blocking (waiting for the build to finish)

Reply:
Does that mean that if events keep on firing very quickly, after say 6ms one will fire, then after e.g. 7ms another one will etc?

Reply:
suppose we have a file event per ms.
0 1 2 3 4 5 6 7 8 9 10
0 --> picked up
1 --> ignored
....
5 --> picked up
...
10 --> picked up